examples/a3c/envs.py

```diff
@@ -10,29 +10,52 @@
 import numpy as np
 import time
 
-import vectorized
-from vectorized.wrappers import Unvectorize, Vectorize
-
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
-def create_env(env_id, client_id, remotes, **kwargs):
-    return create_atari_env(env_id)
-
-def create_atari_env(env_id):
+def create_env(env_id):
     env = gym.make(env_id)
-    env = Vectorize(env)
-    env = AtariRescale42x42(env)
-    env = DiagnosticsInfo(env)
-    env = Unvectorize(env)
+    env = AtariProcessing(env)
+    env = Diagnostic(env)
     return env
 
-def DiagnosticsInfo(env, *args, **kwargs):
-    return vectorized.VectorizeFilter(env, DiagnosticsInfoI, *args, **kwargs)
+def _process_frame42(frame):
+    frame = frame[34:34+160, :160]
```

Comment:
should probably be formatted as `frame[34:(34 + 160), :160]` although i'm not 100% sure about the best way to do this one.